src/global/types.h

```diff
@@ -1555,6 +1555,28 @@ typedef struct CREATURE_INFO {
     PHD_VECTOR target;
 } CREATURE_INFO;
 
+typedef enum {
+    TS_HEADING = 0,
+    TS_BACKGROUND = 1,
+    TS_REQUESTED = 2,
+} TEXT_STYLE;
+
+typedef enum {
+    MC_PURPLE_C = 0,
+    MC_PURPLE_E = 1,
+    MC_BROWN_C = 2,
+    MC_BROWN_E = 3,
+    MC_GREY_C = 4,
+    MC_GREY_E = 5,
+    MC_GREY_TL = 6,
+    MC_GREY_TR = 7,
+    MC_GREY_BL = 8,
+    MC_GREY_BR = 9,
+    MC_BLACK = 10,
+    MC_GOLD_LIGHT = 11,
+    MC_GOLD_DARK = 12,
```

Comment:
In general we should only declare numbers for things that are defined in .PHD etc.
(For T3M it's a different story because our code operates with yet-to-be-decompiled functions that use immutable constants.)